src/printer.js

```diff
@@ -758,10 +762,10 @@ function genericPrintNoParens(path, options, print) {
 
       var isParentForLoop = namedTypes.ForStatement.check(parentNode) ||
         namedTypes.ForInStatement.check(parentNode) |
```

Comment:
Happened to see this, this looks a like a mistake? `namedTypes.ForInStatement.check(parentNode) |` Shouldn't that be `||`?